luigi/contrib/postgres.py

```diff
@@ -364,6 +364,8 @@ class PostgresQuery(rdbms.Query):
     To customize the query signature as recorded in the database marker table, override the `update_id` property.
     """
 
+    port = None
```

Comment:
Why is `port` not defined by the base class? https://github.com/spotify/luigi/blob/master/luigi/contrib/rdbms.py#L263

Reply:
OK host includes both hostname and port.

Reply:
We should handle this similarly like: https://github.com/dhuckins/luigi/blob/de80e083c09221652420cc643d2cd817fff6028e/luigi/contrib/postgres.py#L128

Also please update the docstring to include `port`.

Actually I would prefer we revert https://github.com/spotify/luigi/pull/2615 because it is not needed. Encoding port number as part of `host` would achieve the same, right?

Reply:
yes, looks like that indeed

Reply:
Or we can define `port` property in `rdbms.Query`, which seems more consistent to me. Something like: https://github.com/spotify/luigi/pull/2561/files#diff-1e59aaa5da6736162dabbdec5c5e17c2R185